### PR TITLE
xattr: add cap_sys_ptrace to supported capabilities

### PR DESCRIPTION
**Pull request by jean-edouard**
(Will be) needed for https://github.com/kubevirt/kubevirt/pull/7103

### CODE REVIEW

File: pkg/xattr/xattr.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ const (
 var cap_empty_bitmask = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
 var supported_capabilities = map[string][]byte{
 	"cap_net_bind_service": {1, 0, 0, 2, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+	"cap_sys_ptrace":       {1, 0, 0, 2, 0, 0, 8, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
```
Comment (rmohr) [inline pkg/xattr/xattr.go:17]: Do you think you can add a minimal tar file which contains files with these attrs, load it and check if it matches?

Comment (jean-edouard) [reply pkg/xattr/xattr.go:17]: Do you mean as a functional test? I'm not sure how to do that in here.

Comment (rmohr) [reply pkg/xattr/xattr.go:17]: There is a test in xattr_test.go already which reads a checked in tar archive. That archive can just container handcrafted files via system tools. We can then compare our results to it (like done for the selinux part there).

**Comment by rmohr** [comment]
@jean-edouard @xpivarc what do you think? Added tests and also cap_chown support.

**Comment by xpivarc** [comment]
Looks good to me. It covers both my and Jed's use case.

**Comment by rmohr** [comment]
Thanks @jean-edouard for the PR. Thanks @xpivarc for having a look.

**Pull request merged**